driver/src/main/java/oracle/nosql/driver/values/ArrayValue.java

```diff
@@ -609,4 +617,15 @@ public long sizeof() {
 
         return size;
     }
+
+    /**
+     * Sorts the elements of the array according to the given comparator.
+     *
+     * @param comparator The Comparator to use for comparing the array
+     * elements. It must not be null.
+     */
+    public void sort(Comparator<FieldValue> comparator) {
```

Comment:
Do we want to hide this method for internal use only?

Reply:
Done.